src/main/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -1366,6 +1370,21 @@ private void validateGenotypes() {
         }
     }
 
+    //no controls and white-spaces characters, no semicolon.
+    static final public Pattern VALID_FILTER = Pattern.compile("^[!-:<-~]+$");
+    private void validateFilters() {
+        final Set<String> filters = this.getFilters();
+        if (filters == null) return;
+
+        for (String filter : filters) {
+            if (!VALID_FILTER.matcher(filter).matches()){
+                throw new IllegalStateException("Filter '" + filter +
+                        "' contains an illegal character. It must conform to the regex ;'" + VALID_FILTER.toString());
```

Comment:
unnecessary `toString()`
```suggestion
                        "' contains an illegal character. It must conform to the regex ;'" + VALID_FILTER);
```